### DISCUSSION TITLE
ScriptReference/GeometryUtility.TestPlanesAABB

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#232] [Notes]
# ScriptReference/GeometryUtility.TestPlanesAABB

https://docs.unity3d.com/ScriptReference/GeometryUtility.TestPlanesAABB.html

**Comment by mvi** [comment]
Note this will return some false positives when the far plane intersects along with one or two of the side planes even when the frustum faces do not intersect and the frustum does not intersect the AABB. The size of this false positive will depend on the size of the AABB and is likely not too big an issue for most usecases.  
Also the code sample will return a null ref due to obj always being null. Changing the `obj.name` to `objCollider.name` will resolve the error.
